src/toil/common.py

```diff
@@ -520,6 +522,11 @@ def _addOptions(addGroupFn, config):
     addOptionFn("--retryCount", dest="retryCount", default=None,
                 help=("Number of times to retry a failing job before giving up and "
                       "labeling job failed. default=%s" % config.retryCount))
+    addOptionFn("--enableUnlimitedPreemptableRetries", dest="enableUnlimitedPreemptableRetries",
```

Comment:
Maybe this should be `--preemptibleRetryCount` and accept a value of `inf` instead?

Reply:
This is the only one I haven't changed: mainly b/c it's a lot of work to actually implement proper counting and the specs for having such a value aren't clear (do we start using normal instances after that count or just fail?), and using a float for this felt awkward.
I figured when/if we have the proper implementation for counting failures, we could make it so that setting this bool sets the retry count to a large value and/or deprecate it.
Anyhow lemme know if you feel strongly about it and I can change it :)

Reply:
I think this is fine, if we can't actually count it. We just have to be careful about jobs that manage to take down their machines; we'll retry a `shutdown -h now` forever.

Reply:
hmm...interesting, since we run everything in docker on Mesos, it presumably shouldn't allow that level of access to the instance (although, not sure what happens if a task manages to crash our docker; would it get the LOST status or ERROR?). Anyhow, I think that risk should be pretty low at least on AWS/Mesos side.